packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -70,8 +69,7 @@ contract OptimisticGovernor is Module, Lockable {
         uint256 requestTime;
     }
 
-    mapping(uint256 => bytes32) public proposalHashes;
-    uint256 public prevProposalId;
+    mapping(bytes32 => uint256) public proposalHashes;
```

Comment:
nit: I would either add a comment indicating that this maps to timestamps or change the name of the mapping to proposalTimestamps.